README.md

```diff
@@ -70,7 +88,9 @@ The DataLab job server is deployed to our `dokku2` instance, instructions are ar
 Run `make dev-config` if you have not already. Note: you will need the
 bitwarden cli tool installed in order to access passwords.
 
-A Django development server can be started with `docker-compose up`.
+Run the tests with:
+
+    pytest tests
```

Comment:
There's a makefile target for this, `test`, which runs both coverage and collectstatic.  It's what CI uses and I think we should push for that one by default as it's what's required to get the green ticks for CI.